### ISSUE TITLE
Add shortcut lastweek, month, lastmonth

### ISSUE THREAD

**Issue by valkum** [hardliner66/timetracking#13]
Would be nice if there is a shortcut to get the work time for lastweek without specifying the dates. The same for month and lastmonth.
Makes it easier to make decision without exporting the data.

**Comment by hardliner66** [comment]
Good idea. I'm just not sure if we should make a generalized version (where you could also specify something like: "3 weeks ago") or just some pre-defined filters.
